### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by igor-stasiv**
[DEMO LINK](https://igor-stasiv.github.io/react_goods-selector/)

### CODE REVIEW

**Review by IvanGrekov — APPROVED** [review]
Great solution

File: src/App.js [hunk 1]
```diff
@@ -14,11 +16,97 @@ const goodsFromServer = [
...
+class App extends React.Component {
+  state = {
+    selectedGoods: [],
+  }
+
+  componentDidMount() {
+    this.setState({ selectedGoods: ['Jam'] });
+  }
+
+  setTitleText = () => {
```
Comment (varseniuk) [inline src/App.js:28]:
```suggestion
  getTitleText = () => {
```
When we get some text from the function, it's usually starts with `get`

File: src/App.js [hunk 2]
```diff
@@ -14,11 +16,97 @@ const goodsFromServer = [
...
+          <span className="title__text">
+            {this.setTitleText()}
+          </span>
+
+          <button
+            type="button"
+            className={classNames(`title__button`, {
+              clicked: !selectedGoods.length,
+            })}
+            onClick={() => this.clearSelectedGoods()}
```
Comment (varseniuk) [inline src/App.js:75]:
```suggestion
            onClick={this.clearSelectedGoods}
```
A bit shorter

**Review by varseniuk — APPROVED** [review]
Great job! 👍

**Pull request closed without merging**